master/reference/advanced/etcd-rbac/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Setting up etcd certificates for RBAC
+---
+
+## Overview
+
+When using etcd it is a good idea to protect the data stored there.  This is
+even more true when you have multiple components using a common etcd cluster.
+This guide and the linked guides configure etcd and the components accessing
+etcd with certificates that are linked to etcd RBAC roles to both secure the
+commmunications with the server but also restrict access of each component to
+only the paths it needs.
+
+## Requirements
+
+- Etcd version 3.0.12 is the mimumum version when components will only be using
+  the etcd v2 API, if using the etcd v3 API then etcd version 3.2 is the
```

Comment:
"using v3 API" is likely to cause confusion - you might need to elaborate there since it is only Kubernetes that this would impact I believe.